README.md

```diff
@@ -188,6 +189,13 @@ export type Endpoints = keyof paths;
 
 For more details on how to use see [openapi-typescript documentation](https://openapi-ts.pages.dev/introduction)
 
+## Documentation
+
+For documentations, please visit [our website](https://gsoft-inc.github.io/wl-openapi-typescript/).
+
+The documentation is automatically generated on each release from the files in the docs directory.
+
+
```

Comment:
[Question] @tjosepo As we have the examples in the documentation website, I think we can remove the "Usages" part from here. What do you think? /cc @patricklafrance

Reply:
Yeah, the README.md definitely needs a cleanup. I think it should still include a Getting Started (for users) and a Developing (for contributers) section to the file.

Otherwise, I'm fine with just telling people to check our doc on Retype.

Reply:
Let's get back to it later